packages/jellyfish-api-core/__tests__/category/wallet.test.ts

```diff
@@ -2,27 +2,413 @@ import { ContainerAdapterClient } from '../container_adapter_client'
+
+    it('should createWallet with disablePrivateKeys true', async () => {
+      return await waitForExpect(async () => {
+        const wallet = await client.wallet.createWallet('bob', true)
+
+        expect(wallet.name).toStrictEqual('bob')
+        expect(wallet.warning).toStrictEqual('Empty string given as passphrase, wallet will not be encrypted.')
+      })
+    })
+
+    it('should createWallet with blank true', async () => {
+      return await waitForExpect(async () => {
+        const options = { blank: true }
+        const wallet = await client.wallet.createWallet('charlie', false, options)
+
+        expect(wallet.name).toStrictEqual('charlie')
+        expect(wallet.warning).toStrictEqual('Empty string given as passphrase, wallet will not be encrypted.')
+      })
+    })
+
+    it('should createWallet with passphrase', async () => {
+      return await waitForExpect(async () => {
+        const options = { passphrase: 'shhh...' }
+        const wallet = await client.wallet.createWallet('david', false, options)
+
+        expect(wallet.name).toStrictEqual('david')
+        expect(wallet.warning).toStrictEqual('')
+      })
+    })
+
+    it('should createWallet with avoidReuse true', async () => {
+      return await waitForExpect(async () => {
+        const options = { avoidReuse: true }
+        const wallet = await client.wallet.createWallet('eve', false, options)
+
+        expect(wallet.name).toStrictEqual('eve')
+        expect(wallet.warning).toStrictEqual('Empty string given as passphrase, wallet will not be encrypted.')
+      })
+    })
+  })
+})
+
+describe('sendMany', () => {
+  // NOTE(surangap): defid side(c++) does not have much tests for sendmany RPC atm.
+  const container = new MasterNodeRegTestContainer()
   const client = new ContainerAdapterClient(container)
 
   beforeAll(async () => {
     await container.start()
     await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+    await container.waitForWalletBalanceGTE(101)
   })
 
   afterAll(async () => {
     await container.stop()
   })
 
-  describe('getBalance', () => {
-    it('should getBalance = 0', async () => {
-      const balance: BigNumber = await client.wallet.getBalance()
+  // Returns matching utxos for given transaction id and address.
+  const getMatchingUTXO = async (txId: string, address: string): Promise<UTXO[]> => {
+    const options: ListUnspentOptions = {
+      addresses: [address]
+    }
 
-      expect(balance.toString()).toStrictEqual('0')
+    const utxos: UTXO[] = await client.wallet.listUnspent(1, 9999999, options)
+    return utxos.filter((utxo) => {
+      return (utxo.address === address) && (utxo.txid === txId)
     })
+  }
+
+  it('should send one address using sendMany', async () => {
+    const amounts: Record<string, number> = { mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU: 0.00001 }
+    const transactionId = await client.wallet.sendMany(amounts)
+    expect(typeof transactionId).toBe('string')
```

Comment:
I had changed all toBe to toStrictEqual in JellyFish project. Probably you need to change to toStrictEqual too.